src/pages/Groups/Group1/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,266 @@
+  /* Polar Green/3 */
+  border: 1px solid #b7eb8f;
+  box-sizing: border-box;
+  border-radius: 5px;
+`
+export const CentreNav = styled.div`
+  display: inline-block;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
+  height: 84%;
+  width: 100%;
+  top: 84px;
+  display: flex;
+  justify-content: safe-start;
+  gap: 0px;
+  align-items: center;
+  position: absolute;
+  background: white;
+`
+export const OptionsBoxes = styled.div`
+  display: inline-block;
+  height: 70px;
+  padding: 0;
+  margin: 5px;
+  width: 595px;
+  font-size: 28px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  background: white;
+`
+export const GreyBoxDelete = styled.div`
+  height: 6vh;
+  width: 200px;
+  font-size: 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  position: relative;
+  background: #cecece;
+  border-radius: 10px;
+`
+export const GreyBoxSort = styled.div`
+  width: 100%;
+  height: 6vh;
+  width: 200px;
+  font-size: 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  position: relative;
+  background: #cecece;
+  border-radius: 10px;
+`
+export const GreyBoxText = styled.p<{ fontSize?: string }>`
+  ${(props) => props.fontSize && `font-size: ${props.fontSize}`}
+  font-family: Lato;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  height: 0vh;
+  top: 10px;
+  line-height: 0px;
+  color: rgba(25, 25, 25, 1);
+  justify-content: space-between;
+  position: relative;
+  align-items: stretch;
+`
+export const StyledGreybox = styled.img`
+  width: 28px;
+  position: relative;
+`
+export const Booking = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px 26px;
+  position: relative;
+  width: 600px;
+  margin: 15px 2%;
+  height: 160px;
+  background: #ffffff;
+  border: 1px solid #b7eb8f;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 5px;
+`
+export const BookingLocation = styled.p<{ fontSize?: string }>`
+  ${(props) => props.fontSize && `font-size: ${props.fontSize}`}
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 25px;
+  line-height: 22px;
+  position: absolute;
+  left: 150px;
+  color: rgba(0, 0, 0, 0.85);
+`
+export const BookingCCA = styled.p<{ fontSize?: string }>`
+  ${(props) => props.fontSize && `font-size: ${props.fontSize}`}
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 110px;
+  position: absolute;
+  left: 150px;
+  color: rgba(0, 0, 0, 0.65);
+`
+export const BookingTiming = styled.p<{ fontSize?: string }>`
+  ${(props) => props.fontSize && `font-size: ${props.fontSize}`}
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 200px;
```

Comment:
Way too much, cover other components. Same for a lot of other components.

<img width="432" alt="Screenshot 2021-10-09 at 11 43 29 PM" src="https://user-images.githubusercontent.com/64590253/136665202-95d71e0a-9ef6-4bfe-a124-8d9ed0b20e6a.png">
Image shows what happen when i try to highlight that text.